Gemfile.lock

```diff
@@ -16,3 +16,6 @@ DEPENDENCIES
   attr_comparable!
   minitest
   rake (>= 0.9)
+
+BUNDLED WITH
+   1.12.5
```

Comment:
This was an issue before, but looks like at some point we moved to including it in web late last year: https://github.com/Invoca/web/blob/master/Gemfile.lock#L1340-L1341

And I think that started when we moved over to bundler 1.12+ 
https://github.com/Invoca/laptop/blob/2ce5201101ebf0f802bf31e67289c170f32a6214/common-components/ruby-environment#L32

So likely this is fine / preferred to leave in here at this point?